samples/release-management/csharp/README.md

```diff
@@ -0,0 +1,151 @@
+---
+page_type: sample
+description: This is a sample app with capability to send notfication when user creates workitem in [Azure DevOps](https://dev.azure.com) via webhooks.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- csharp
+extensions:
+contentType: samples
+createdDate: "29-04-2022 00:02:15"
+---
+
+# Bot to created group chat and send task notfication using Azure webhooks.
```

Comment:
Bot to create the group chat and send task notification using Azure webhooks.

Reply:
Done